packages/quick_actions/quick_actions_android/example/android/app/src/androidTest/java/io/flutter/plugins/quickactionsexample/QuickActionsTest.java

```diff
@@ -20,4 +54,108 @@ public void imagePickerPluginIsAdded() {
           assertTrue(activity.engine.getPlugins().has(QuickActionsPlugin.class));
         });
   }
+
+  @Test
+  public void appShortcutsAreCreated() {
+    // Arrange
```

Comment:
I appreciate the organization of the tests, but can you remove the "arrange", "act", and "assert" comments? I think the clarity will remain!

Reply:
Sure.